api/Window.json

```diff
@@ -50,6 +50,161 @@
           "deprecated": false
         }
       },
+      "alert": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/alert",
+          "support": {
+            "chrome": {
+              "version_added": true,
+              "notes": "Starting with Chrome 46.0 this method is blocked inside an <code>&lt;iframe&gt;</code> unless its sandbox attribute has the value <code>allow-modals</code>."
+            },
+            "chrome_android": {
+              "version_added": true
```

Comment:
The Chrome note applies to chorme_android, android_webview, and both flavors of Opera.